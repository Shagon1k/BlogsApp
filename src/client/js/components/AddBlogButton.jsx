import React from 'react';

const AddBlogButton = ({onBlogAdd}) => (
	<button className="addBlogBtn btn" onClick={() => onBlogAdd()} >Add blog</button>
)

export default AddBlogButton;