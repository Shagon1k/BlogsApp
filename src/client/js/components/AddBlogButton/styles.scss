.addBlogBtn {
	width: 65%;
	height: 45px;
	border-radius: 15px;
}