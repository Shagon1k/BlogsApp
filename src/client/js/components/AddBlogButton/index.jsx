import React from 'react';
import { Link } from 'react-router-dom';
import './styles.scss';

const AddBlogBtn = () => (
	<Link to="/blogs/add" className="addBlog btn">Add blog</Link>
)

export default AddBlogBtn;