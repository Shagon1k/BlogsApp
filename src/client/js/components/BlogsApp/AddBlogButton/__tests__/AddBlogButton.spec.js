import React from 'react';
import AddBlogButton from '../index.jsx';

describe('AddBlogButton component', () => {
	it('renders correctly', () => {
		const wrapper = shallow(<AddBlogButton/>);
		expect(wrapper).toMatchSnapshot();
	});
})