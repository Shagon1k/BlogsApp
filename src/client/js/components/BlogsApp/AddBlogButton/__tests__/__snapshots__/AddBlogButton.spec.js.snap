// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddBlogButton component renders correctly 1`] = `
<Link
  className="addBlog btn"
  replace={false}
  to="/blogs/add"
>
  Add blog
</Link>
`;
